graphql/src/schema/api.rs

```diff
@@ -643,32 +655,36 @@ fn collection_arguments_for_named_type(type_name: &str) -> Vec<InputValue> {
     let mut first = input_value(&"first".to_string(), "", Type::NamedType("Int".to_string()));
     first.default_value = Some(Value::Int(100.into()));
 
-    let args = vec![
+    let mut args = vec![
         skip,
         first,
         input_value(
+            &"where".to_string(),
+            "",
+            Type::NamedType(format!("{}_filter", type_name)),
+        ),
+    ];
+
+    if version.supports(FeatureFlag::BasicOrdering) {
```

Comment:
I am losing track of the plot a little here - should we really have a version without ordering?

Reply:
No no, it's just an example of how to use the feature flags
https://github.com/graphprotocol/graph-node/pull/3185#discussion_r791501883